client/components/widget/data_viz/gviz/column_style/column-style-service.js

```diff
@@ -179,10 +196,16 @@ gviz.column_style.ColumnStyleService = class {
       let columnIndex = this.getColumnIndex(column.column_id, dataTable);
 
       if (columnIndex !== -1) {
-        if (!goog.string.isEmptySafe(column.title)) {
+        if (goog.string.isEmptySafe(column.title)) {
+          dataTable.setColumnLabel(columnIndex, column.column_id);
```

Comment:
Not necessary for this checkin, but at some point it'd be interesting to normalize the metadata names and property names (e.g. setColumnTitle(column.title) is easier to read. The same for setColumnRole(column.role))